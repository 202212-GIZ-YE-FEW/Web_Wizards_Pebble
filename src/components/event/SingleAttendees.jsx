const SingleAttendees = ({ name }) => {
    const firstLetter = name && name.charAt(0).toUpperCase();

    return (
        <div className='py-2 w-1/3 md:w-1/6 lg:w-1/4'>
            <div className='flex items-center justify-center w-16 h-16 rounded-full bg-black-100 text-white text-md font-bold mb-1'>
                <span>{firstLetter}</span>
            </div>
            <p className='text-sm font-semibold'>{name}</p>
        </div>
    );
};

export default SingleAttendees;
