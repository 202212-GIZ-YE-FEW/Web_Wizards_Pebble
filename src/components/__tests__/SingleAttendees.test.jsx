import renderer from "react-test-renderer";

import SingleAttendees from "@/components/event/SingleAttendees";

it("renders correctly", () => {
    const tree = renderer.create(<SingleAttendees />).toJSON();
    expect(tree).toMatchSnapshot();
});
