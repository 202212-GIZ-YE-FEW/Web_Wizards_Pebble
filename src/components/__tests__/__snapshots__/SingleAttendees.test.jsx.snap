// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="py-2 w-1/3 md:w-1/6 lg:w-1/4"
>
  <div
    className="flex items-center justify-center w-16 h-16 rounded-full bg-black-100 text-white text-md font-bold mb-1"
  >
    <span />
  </div>
  <p
    className="text-sm font-semibold"
  />
</div>
`;
